src/components/address/address.tsx

```diff
@@ -0,0 +1,31 @@
+import { useResolveEnsName } from 'eth-hooks/dapps';
+import { TEthersProvider } from 'eth-hooks/models';
+
+export interface AddressProps {
```

Comment:
Can we prefix interfaces with `I` and types with `T`.  it was the standard in eth hooks